op-service/signer/cli.go

```diff
@@ -27,6 +30,11 @@ func CLIFlags(envPrefix string) []cli.Flag {
 			Usage:   "Address the signer is signing transactions for",
 			EnvVars: opservice.PrefixEnvVar(envPrefix, "ADDRESS"),
 		},
+		&cli.StringSliceFlag{
+			Name:    HeadersFlagName,
+			Usage:   "Request headers",
```

Comment:
It would be good to have an example here of what input looks like

Reply:
Explanation added